crit/explore.go

```diff
@@ -24,6 +24,20 @@ type PsTree struct {
 	Children []*PsTree            `json:"children,omitempty"`
 }
 
+// FindPs performs a short-circuiting depth-first search to find
+// a process with a given PID in a process tree.
+func (c *PsTree) FindPs(pid uint32) *PsTree {
```

Comment:
```suggestion
func (ps *PsTree) FindPs(pid uint32) *PsTree {
```
Also, I feel this function would be better off in `utils.go`, since the functionality is not related to `crit explore`.